CAM2CameraDatabaseAPIClient/camera.py

```diff
@@ -2,9 +2,12 @@
 Represents a camera.
 """
 
-class Camera(object):
+class Camera(dict):
 
     """Class representing a general camera.
+    It inherits from dict and uses  all its methods as defined in dict.
+    Therefore, there's no need to define a constructor.
+
```

Comment:
this doc string will be included in the documentation page, you can refer to sphinx numpy documentation style and place it somewhere else where it will not be included in the documentation and only for developers to reference.

Reply:
Done I'm communicating with @sripathm22  to have it on the wiki instead